docs/v8-migration-api.md

```diff
@@ -77,6 +77,38 @@ Check out the [v8-migration guide](v8-migration.md) for other user-level changes
     - For `node.parameters` in Babel 7, use `node.params` in Babel 8
     - For `node.typeAnnotation` in Babel 7, use `node.returnType` in Babel 8
 
+- Rename `typeParameters` to `typeArguments` for `TSTypeReference` ([#16679](https://github.com/babel/babel/issues/16679), [#17008](https://github.com/babel/babel/pull/17008))
+
+  ```ts title=input.ts
+  var arr: Array<string>;
+
+  // AST in Babel 7
+  {
+    type: "TSTypeReference",
+    typeName: Identifier("Array"),
+    typeParameters: {
+      type: "TSTypeParameterInstantiation",
+      params: [{
+        type: "TSStringKeyword"
+      }]
+    }
+  }
+
+  // AST in Babel 8
+  {
+    type: "TSTypeReference",
+    typeName: Identifier("Array"),
+    typeArguments: {
+      type: "TSTypeParameterInstantiation",
+      params: [{
+        type: "TSStringKeyword"
+      }]
+    }
+  }
+  ```
+
+  **Migration**: If you have a customized plugin accessing `typeParameters`, use `typeArguments` in Babel 8
```

Comment:
Sometimes the migration seems so obvious that probably it doesn't make sense to write it explicitly 😅 (not just here, but in many other AST changes).

If we already say "we renamed `foo` to `bar`", it's obvious that the migration is to rename `foo` to `bar`.